### PR TITLE
Update french translation

### PR DESCRIPTION
**Pull request by Nilvus**
French translation update. @aurelienpierre and @TurboGit: if you could check all is good for you? Thanks.

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -17301,273 +17614,347 @@ msgctxt "accel"
...
 
-#: ../src/libs/modulegroups.c:1750
+#: ../src/libs/modulegroups.c:2763
 msgid "technical icon"
 msgstr "icône technique"
 
-#: ../src/libs/modulegroups.c:1775
+#: ../src/libs/modulegroups.c:2795 ../src/libs/modulegroups.c:2797
+msgid "basics widgets"
+msgstr "widgets basiques"
```
Comment (TurboGit) [inline po/fr.po:17886]: widgets -> contrôles ?

File: po/fr.po [hunk 2]
```diff
@@ -17301,273 +17614,347 @@ msgctxt "accel"
...
+
+#: ../src/libs/modulegroups.c:2818
+msgid "add widgets to the list"
+msgstr "ajoute un widget à la liste"
```
Comment (TurboGit) [inline po/fr.po:17890]: widget -> contrôle ?

File: po/fr.po [hunk 3]
```diff
@@ -17301,273 +17614,347 @@ msgctxt "accel"
...
 msgstr[1] "modules déjà présents"
 
-#. we show the list of recommended modules
-#: ../src/libs/modulegroups.c:1467
-msgid "recommended module"
-msgid_plural "recommended modules"
-msgstr[0] "module recommandé"
+#: ../src/libs/modulegroups.c:2243 ../src/libs/modulegroups.c:2297
+msgid "click to remove the widget"
+msgstr "cliquer pour supprimer le widget"
```
Comment (TurboGit) [inline po/fr.po:17796]: widget -> contrôle ?

File: po/fr.po [hunk 4]
```diff
@@ -17301,273 +17614,347 @@ msgctxt "accel"
...
+
+#: ../src/libs/modulegroups.c:2258 ../src/libs/modulegroups.c:2266
+#: ../src/libs/modulegroups.c:2312 ../src/libs/modulegroups.c:2320
+msgid "click to add the widget"
+msgstr "cliquer pour ajouter le widget"
```
Comment (TurboGit) [inline po/fr.po:17801]: widget -> contrôle ?

File: po/fr.po [hunk 5]
```diff
@@ -17301,273 +17614,347 @@ msgctxt "accel"
...
+
+#: ../src/libs/modulegroups.c:2353
+msgid "recommended widget"
+msgid_plural "recommended widgets"
+msgstr[0] "widget recommandé"
```
Comment (TurboGit) [inline po/fr.po:17806]: widget -> contrôle

File: po/fr.po [hunk 6]
```diff
@@ -17301,273 +17614,347 @@ msgctxt "accel"
...
 
-#: ../src/libs/modulegroups.c:1120
+#: ../src/libs/modulegroups.c:1605
 msgid "previous config with new layout"
 msgstr "configuration précédente avec une nouvelle disposition"
 
-#: ../src/libs/modulegroups.c:1267
+#: ../src/libs/modulegroups.c:1743
+msgid "remove this widget"
+msgstr "supprimer ce widget"
```
Comment (TurboGit) [inline po/fr.po:17751]: widget -> contrôle

File: po/fr.po [hunk 7]
```diff
@@ -17301,273 +17614,347 @@ msgctxt "accel"
...
-#: ../src/libs/modulegroups.c:1048 ../src/libs/modulegroups.c:1063
+#: ../src/libs/modulegroups.c:663
+msgid "(some features may only be available in the full module)"
+msgstr ""
+"(certaines fonctionnalités ne sont disponibles que dans le module complet)"
+
+#: ../src/libs/modulegroups.c:690
+#, c-format
+msgid "go to full version of module %s"
+msgstr "Aller au module complet %s"
```
Comment (TurboGit) [inline po/fr.po:17664]: pas de majuscule

File: po/fr.po [hunk 8]
```diff
@@ -17301,273 +17614,347 @@ msgctxt "accel"
...
-#: ../src/libs/modulegroups.c:296
-msgid "clear text"
-msgstr "effacer texte"
+#: ../src/libs/modulegroups.c:545 ../src/libs/modulegroups.c:547
+#: ../src/libs/modulegroups.c:650
+msgid ""
+"This basic widget is disabled as there's multiple instances for this module. "
+"You need to use the full module..."
+msgstr ""
+"Ce widget de base est désactivé car il y a plusieurs instances pour ce "
```
Comment (TurboGit) [inline po/fr.po:17645]: pas de majuscule (je corrige sur l'anglais)

**Review by TurboGit — CHANGES_REQUESTED** [review]
Difficult to review as many diffs are due to change in order of messages!!! Many diffs are not actual changes...

I have spotted at least widget used in many places which could well be replaced by **contrôle** as we are speaking about a specific control in a module.

**Comment by Nilvus** [comment]
> Difficult to review as many diffs are due to change in order of messages!!! Many diffs are not actual changes...
> 
> I have spotted at least widget used in many places which could well be replaced by **contrôle** as we are speaking about a specific control in a module.

widget is used in french, that's why I don't change them. I'm for keeping them but I'm ok too for "contrôle" instead.

File: po/fr.po (hunk 7 above)
Comment (Nilvus) [reply po/fr.po:17664]: Je n'ai pas traduit ça mais je corrige.

File: po/fr.po (hunk 8 above)
Comment (Nilvus) [reply po/fr.po:17645]: Ok

**Comment by Nilvus** [comment]
> Difficult to review as many diffs are due to change in order of messages!!! Many diffs are not actual changes...
> 
> I have spotted at least widget used in many places which could well be replaced by **contrôle** as we are speaking about a specific control in a module.

On that, I don't know how to do differently. I had just updated pot and then fr.po to make that translation.

**Comment by TurboGit** [comment]
> On that, I don't know how to do differently.

You cannot do differently, not your fault that the way updating a po file is done.

**Comment by Nilvus** [comment]
@TurboGit: new colorbalance rgb seems to not be integrated for translation. I've just see it in english in master and I don't see related text in po.fr, after updating it of course. Could you check what Is missing and ping me? I will update that PR after.

**Comment by TurboGit** [comment]
> @TurboGit: new colorbalance rgb seems to not be integrated for translation. I've just see it in english in master and I don't see related text in po.fr, after updating it of course. Could you check what Is missing and ping me? I will update that PR after.

Indeed, it was missing. I have just added. Would it makes sense to merge this PR and work on colorbalancergb into another PR ? It will be easier to review. Anyway is fine by me, just let me know.

**Comment by Nilvus** [comment]
Ok,  i will do another PR. You could merge that one.

**Comment by TurboGit** [comment]
Good, merging then. Thanks.

**Pull request merged**